output-management/web-viewer/api-basics/java/src/main/java/com/ca/om/webviewer/samples/api/Demo.java

```diff
@@ -0,0 +1,474 @@
+    CommandLine commandLine = parser.parse(options, args);
+
+    if (commandLine.hasOption(HELP)) {
+      HelpFormatter formatter = new HelpFormatter();
+      formatter.setOptionComparator(null);
+      formatter.printHelp("CLITester", options);
+      return;
+    }
+
+    String userName = null;
+    String password = null;
+    String basePath = null;
+
+    // Check for each required parameter.
+    if (!(commandLine.hasOption(BASE_PATH) && commandLine.hasOption(USER_NAME) && commandLine
+        .hasOption(USER_PASSWORD))) {
+      System.out.println(
+          "Missing required parameters: " + BASE_PATH + "," + USER_NAME + "," + USER_PASSWORD);
+      return;
+    }
+
+    if (commandLine.hasOption(BASE_PATH)) {
+      basePath = commandLine.getOptionValue(BASE_PATH);
+    }
+
+    if (commandLine.hasOption(USER_NAME)) {
+      userName = commandLine.getOptionValue(USER_NAME);
+    }
+
+    if (commandLine.hasOption(USER_PASSWORD)) {
+      password = commandLine.getOptionValue(USER_PASSWORD);
+    }
+
+    // Default operation: List report content without applying indexing filters.
+    if (commandLine.hasOption(OPERATION)) {
+      int operation = Integer.parseInt(commandLine.getOptionValue(OPERATION));
+      switch (operation) {
+        case OPERATION_AUTHENTICATION:
+          loginAndLogout(basePath, userName, password);
+          break;
+        case OPERATION_DISPLAY_REPORT:
+          listReportContent(basePath, userName, password, commandLine);
+          break;
+        case OPERATION_DISPLAY_FILTERED_REPORT:
+          listCrossIndexReports(basePath, userName, password, commandLine);
+          break;
+        default:
+          listReportContent(basePath, userName, password, commandLine);
+          break;
+      }
+    } else {
+      listReportContent(basePath, userName, password, commandLine);
+    }
+  }
+
+  /**
+   * <p><ul>
+   * <li>Log in with a username and password to obtain a session token.</li>
+   * <li>Log out by invalidating the session token.</li>
+   * </ul></p>
+   *
+   * @param basePath Http address and port for the application server.
+   * @param username User id to authenticate.
+   * @param password User password to authenticate.
+   */
+  private static void loginAndLogout(String basePath, String username, String password) {
+    // Authenticate with username and password.
+    String sessionToken = login(basePath, username, password);
+
+    // Log out by invalidating the session token.
+    if (sessionToken != null) {
+      logout(basePath, sessionToken);
+    }
+  }
+
+
+  /**
+   * Extract command-line arguments for a report list and follow these steps:
+   * <p><ul>
+   * <li>Log in with a username and password to obtain a session token.</li>
+   * <li>List a repository.</li>
+   * <li>List the report for the first found repository id.</li>
+   * <li>Print the content of first found text report.</li>
+   * <li>Log out by invalidating the session token.</li>
+   * </ul></p>
```

Comment:
There were couple fo recent fixes to javadoc that are not reflected in the sample import, see `git diff v14.0.55..development -- samples`